src/Helpers.cpp

```diff
@@ -59,49 +59,49 @@ bool fastDisconnectionManagement = false;
 
 void Helpers::smartPrint(String msg) {
 
-  if (PRINT_TO_DISPLAY) {
+  #if (DISPLAY_ENABLED) 
     display.print(msg);
-  } else {
+  #else
     Serial.print(msg);
-  }
+  #endif
 
 }
 
 void Helpers::smartPrintln(String msg) {
 
-  if (PRINT_TO_DISPLAY) {
-    display.println(msg);
-  } else {
-    Serial.println(msg);
-  }
+  #if (DISPLAY_ENABLED) 
+    display.print(msg);
```

Comment:
please don't change 
    display.println(msg);
into
    display.print(msg);

Reply:
I will do it by myslef since I need to release a new version today :)
Thank you for your contribution.